content/en/news/releases/1.14.x/announcing-1.14.5/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.14.5
+linktitle: 1.14.5
+subtitle: Patch Release
+description: Istio 1.14.5 patch release.
+publishdate: 2022-10-07
```

Comment:
Will need to update this to the correct date (10-11)?

Reply:
Done, thanks!